src/antidote.erl

```diff
@@ -57,6 +57,7 @@
 
 -spec start() -> {ok, [atom()]} | {error, reason()}.
 start() ->
+    ok = application:ensure_started(chumak),
```

Comment:
Is this line still necessary? Normally ensure_all_started should also take care of chumak since it is defined under applications in antidote.app.src
Or is it different here?